user_docs/en/changes.md

```diff
@@ -59,6 +64,9 @@ It is especially useful to read the error location markers in tracebacks. (#1632
 * When a `gainFocus` event is queued with an object that has a valid `focusRedirect` property, the object pointed to by the `focusRedirect` property is now held by `eventHandler.lastQueuedFocusObject`, rather than the originally queued object. (#15843)
 * NVDA will log its executable architecture (x86) at startup. (#16432, @josephsl)
 * `wx.CallAfter`, which is wrapped in `monkeyPatches/wxMonkeyPatches.py`, now includes proper `functools.wraps` indication. (#16520, @XLTechie)
+* There is a new module for scheduling tasks, using the pip module `schedule`.

```

Comment:
I think that some advice to schedule jobs that display messages undefinitely at intervals would be useful. I'm thinking in add-ons like Gestor de Pomodoros, by @jpavonabian . Imo the schedule module maybe helpful, but some suggestions to avoid conflict, or a small example in the developer guide, would be appreciated. For example, what happens if a message is presented each 30 seconds and another job overlaps this?

Reply:
I've improved the function descriptions and API a bit further. Can you let me know your thoughts of what else could change?